salesagility/SuiteCRM
# Fixed #6766 - SMTP Port Issue

## Description



Type was equal to int which changed the value to have a comma in it, the type is now equal to value and thus the comma is no longer there.

## Motivation and Context
Fixed #6766 

## How To Test This
1. add in the details in the outbound server.
2. test before saving - all fine
3. save and test again see the , in the port will now send


## Types of changes


### Final checklist
